modules/dreamview/backend/sim_control/sim_control.cc

```diff
@@ -107,6 +102,56 @@ void SimControl::OnReceiveNavigationInfo(
   }
 }
 
+void SimControl::InitStartPoint(double start_velocity,
+                                double start_acceleration) {
+  TrajectoryPoint point;
+  // Use the latest localization position as start point,
+  // fall back to a dummy point from map
+  AdapterManager::GetLocalization()->Observe();
+  if (AdapterManager::GetLocalization()->Empty()) {
+    start_point_set_ = false;
```

Comment:
can we have a more accurate naming here? Setting start_point_set_  to false seems to contradict with what the code actually does here.